integration/spark2/src/main/scala/org/apache/spark/sql/execution/strategy/StreamingTableStrategy.scala

```diff
@@ -59,7 +59,8 @@ private[sql] class StreamingTableStrategy(sparkSession: SparkSession) extends Sp
           new TableIdentifier(model.tableName, model.databaseName),
           "Alter table change datatype")
         Nil
-      case AlterTableRenameCommand(oldTableIdentifier, _, _) =>
+      case AlterTableRenameCommand(
+      oldTableIdentifier, _, _) if (isCarbonStreamingTable(oldTableIdentifier)) =>
```

Comment:
Why this if check is needed, `rejectIfStreamingTable` checks whether it is streaming table already.
Please describe the bug in PR description, otherwise not sure what bug it is.